src/setup.py

```diff
@@ -23,59 +24,89 @@ def config_logger(logfile_path, logfile_name="medaka_outdir.log", in_debug_mode=
-    parser.add_argument('-m','--maxjobs',   action="store",         dest='maxjobs',     help='maxjobs on the cluster',          default=None,   required = False)
-    parser.add_argument('-x', '--lsf_index',action="store",         dest='lsf_index',   help=argparse.SUPPRESS,                                 required = False)
+    parser.add_argument('--slowmode',       action="store_true",    dest='slowmode',
+                        help='Should run analysis in slowmode',                 required=False)
+    parser.add_argument('--cluster',        action="store_true",    dest='cluster',
+                        help='Run analysis on a cluster',                       required=False)
+    parser.add_argument('--email',          action="store_true",    dest='email',
+                        help='Receive email for cluster notification',          required=False)
+    parser.add_argument('-m', '--maxjobs',   action="store",         dest='maxjobs',
+                        help='maxjobs on the cluster',          default=None,   required=False)
+    parser.add_argument('-x', '--lsf_index', action="store",         dest='lsf_index',
+                        help=argparse.SUPPRESS,                                 required=False)
 
     # Debug flag
-    parser.add_argument('--debug',          action="store_true",    dest='debug',      help='Additional debug output',                          required = False)
-    parser.set_defaults(crop=False, only_crop=False, crop_and_save=False, slowmode=False, cluster=False, email=False, debug=False)
+    parser.add_argument('--debug',          action="store_true",    dest='debug',
+                        help='Additional debug output',                          required=False)
+    parser.set_defaults(crop=False, only_crop=False, crop_and_save=False,
+                        slowmode=False, cluster=False, email=False, debug=False)
     args = parser.parse_args()
 
     # Move up one folder if croppedRAWTiff was given. Experiment folder is above it.
     if os.path.basename(os.path.normpath(args.indir)) == "croppedRAWTiff":
         args.indir = os.path.dirname(os.path.normpath(args.indir))
 
     # Adds a trailing slash if it is missing.
-    args.indir  = os.path.join(args.indir, '')
+    args.indir = os.path.join(args.indir, '')
     if args.outdir:
         args.outdir = os.path.join(args.outdir, '')
 
     return args
 
 # Processing, done after the logger in the main file has been set up
+
+
 def process_arguments(args):
     will_crop = (args.only_crop or args.crop_and_save or args.crop)
 
     # Folder structure
     if not args.outdir:
         args.outdir = args.indir
 
+    # Get the experiment name and verify if there is cropp folder
+    if os.path.basename(os.path.normpath(args.indir)) == "croppedRAWTiff":
+        temp_name = os.path.dirname(os.path.normpath(args.indir))
+    else:
+        temp_name = os.path.basename(os.path.normpath(args.indir))
+
```

Comment:
Variable temp_name is never used after this check?

Reply:
Please, in the same file, change from:

experiment_name = os.path.basename(os.path.normpath(args.indir))

To:

experiment_name = os.path.basename(os.path.normpath(temp_name))